subworkflows/local/bam_variant_calling_haplotypecaller/main.nf

```diff
@@ -101,12 +101,12 @@ workflow BAM_VARIANT_CALLING_HAPLOTYPECALLER {
     versions = versions.mix(GATK4_HAPLOTYPECALLER.out.versions)
     versions = versions.mix(MERGE_HAPLOTYPECALLER.out.versions)
 
-    // add variantcaller to meta map and remove no longer necessary field: num_intervals
-    vcf = vcf.map{ meta, vcf -> [ meta - meta.subMap('num_intervals') + [ variantcaller:'haplotypecaller' ], vcf ] }
+    // Remove no longer necessary field: num_intervals
+    vcf = vcf.map{ meta, vcf -> [ meta - meta.subMap('num_intervals'), vcf ] }
```

Comment:
why did you move the metamap assignment?

Reply:
Good question. The addition of the `variantcaller` tag in the metamap in that line seems to be redundant. The variantcaller tag is set above in the same script (line 34) just before the call to the haplotypecaller. I ran a test where I put a `vcf.view`-statement just after line 105 (the line where the tag `num_intervals` is removed) and I got this:
```
vcf : [[patient:test, sample:test, sex:NA, status:0, id:test, data_type:bam, variantcaller:haplotypecaller], /home/ubuntu/dev/sentieon/work/e9/5ead5cbe9e23f0ae1da3ff52f4895f/test.haplotypecaller.filtered.vcf.gz]
```